src/Symfony/Component/Console/CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ CHANGELOG
  * Add `BackedEnum` support with `#[Argument]` and `#[Option]` inputs in invokable commands
  * Allow Usages to be specified via `#[AsCommand]` attribute.
  * Allow passing invokable commands to `Symfony\Component\Console\Tester\CommandTester`
+ * add optional timeout for human in `QuestionHelper`
```

Comment:
```suggestion
 * Add optional timeout for interaction in `QuestionHelper`
```

Reply:
This one was made by @OskarStark: https://github.com/symfony/symfony/pull/61092#discussion_r2197884077 

Applying yours.